samples/Chat/src/app/utils/ShakeToSendLogs.tsx

```diff
@@ -27,21 +27,24 @@ const storeLog = (logType: string, log: string | undefined): void => {
  * This is particularly useful on mobile devices where the console is not easily accessible.
  */
 const startRecordingLogs = (): void => {
-  function hookLogType(logType: string): (...args: unknown[]) => void {
+  function hookLogType(logType: string, outputToConsole: boolean): (...args: unknown[]) => void {
```

Comment:
Is there a section in the docs about the log mechanisms?

Reply:
There isn't, is there somewhere particular we should enter information about this?
(added a comment to the top of this file also to detail what the purpose of this component is)